setup.py

```diff
@@ -31,6 +31,7 @@ def read(filename):
     package_dir={"": "src"},
     packages=find_packages(where="src", exclude=("tests",)),
     install_requires=[
+        "bokeh",
```

Comment:
Do we need an explicit `bokeh` dependency? I thought `dask` took care of this by itself?

Reply:
Merging anyway, we can remove this dependency later on if we figure out it is redundant...